check_nodes_test.go

```diff
@@ -169,11 +169,36 @@ func TestCheckNodesWithErrors(t *testing.T) {
 	}
 
 	errCollector := newErrorsCollector()
-	checkNodes(context.Background(), nodes, executor, Tracer{}, errCollector)
+	checkNodes(context.Background(), nodes, executor, Tracer{}, &errCollector)
 
 	err := errCollector.Err()
 	for i := 0; i < count; i++ {
-		assert.ErrorContains(t, err, fmt.Sprintf("error on node %s", nodes[i].Addr()))
+		assert.ErrorContains(t, err, fmt.Sprintf("'%s' node error occurred at", nodes[i].Addr()))
```

Comment:
%q is a standard way to quote something in formatting and is preferred to '%s'

Reply:
Fixed.